domain/src/main/java/com/annhienktuit/domain/models/Song.java

```diff
@@ -2,6 +2,7 @@
 
 import com.google.gson.annotations.SerializedName;
 
+//Model
```

Comment:
I think u shouldn't use @SerializedName in domain module, it should be use in data module